R/corrpart.b.R

```diff
@@ -26,26 +25,19 @@ corrPartClass <- R6::R6Class(
             controls <- self$options$controls
             type <- self$options$type
 
-            if (hyp == 'pos')
-                hyp <- 'greater'
-            else if (hyp == 'neg')
-                hyp <- 'less'
-            else
-                hyp <- 'equal'
-
             results <- list()
             if (nVars > 1) {
                 for (i in 1:nVars) {
                     rowVar <- vars[[i]]
                     for (j in 1:nVars) {
-                        if (i == j || (type != 'semi' && j > i))
-                            next
+                        if (j >= i) { next }
```

Comment:
Don't use brackets with inline statements (I would even prefer to avoid inline statements altogether if possible). Maybe have a look at the tidyverse style guide (although we do some things a bit different, e.g., 4 spaces instead of 2): https://style.tidyverse.org/syntax.html#control-flow. There are couple of other places where you do this.

I usually use brackets if the content has two lines or more, and no brackets if the content is only one line, but always put the content on a new line.

So:
```
if (statement == TRUE)
    next
```
Or:
```
if (statement == TRUE) {
    x <- 'assign something'
    return(x)
}
```